### PR TITLE
Vertex AI Experiments - Comparing local trained models notebook - update

### PR DESCRIPTION
**Pull request by inardini**
<br>
This PR has the goal to update Vertex AI Experiment notebook based on collected feedback. 
<br><br><br>

If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/experiments/comparing_local_trained_models.ipynb [hunk 1]
```diff
@@ -32,51 +32,48 @@
         "<table align=\"left\">\n",
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>This should be H1 title (if not already) and moved into the above links cell above the links.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>Separated this into 4 cells.</p><p>Move dataset after Objective.</p><p> </p><p>Add "Vertex AI TensorBoard" to bullet list of billable components.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>Combine these 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>Use the current template version for PROJECT_ID</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>Remove 'Skip this step'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>TensorBoard -&gt; Vertex AI TensorBoard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>we'd like to introduce the following concepts: -&gt; the following concept are explained</p><p> </p><p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>I recommend combining these 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>combine these 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>Set delete_bucket to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:32]: <p>done</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**